.github/workflows/critical-issue.yml

```diff
@@ -8,10 +8,20 @@ jobs:
   slack-notify:
     name: Send slack notification if issue is labeled as critical
     runs-on: ubuntu-latest
-    if: ${{ github.event.label.name == 'severity/critical' }}
```

Comment:
:thought_balloon: The filter can remain on the job level and doesn't need to be moved to the `slack-notify` step, right?

Reply:
Oh yes 👍